Checkstyle rule violations were found.

Checkstyle rule violations were found. See the report at: file:///C:/Users//Downloads/RxAndroidBle-master/rxandroidble/build/reports/checkstyle/checkstyle.html
Checkstyle files with violations: 1
Checkstyle violations by severity: [error:1]

i'm getting this kind of issue please help me i'm new in android

Okay i ran this ` ./gradlew :sample-kotlin:assembleDebug` cause i'm exploring the kotlin sample. And this is what i got: 

> > Task :rxandroidble:checkstyle
> [ant:checkstyle] [ERROR] D:\workspace\GitHub\RxAndroidBle\rxandroidble\src\main\java\com\polidea\rxandroidble2\internal\connection
> \DisconnectionRouter.java:125: Line is longer than 140 characters (found 141). [LineLength]
> 
> > Task :rxandroidble:checkstyle FAILED
> 
> FAILURE: Build failed with an exception.
> 
> * What went wrong:
> Execution failed for task ':rxandroidble:checkstyle'.
> > Checkstyle rule violations were found. See the report at: file:///D:/workspace/GitHub/RxAndroidBle/rxandroidble/build/reports/ch
> eckstyle/checkstyle.html
>   Checkstyle files with violations: 1
>   Checkstyle violations by severity: [error:1]
> 
> 
> * Try:
> Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to
> get full insights.
> 
> * Get more help at https://help.gradle.org
> 
> Deprecated Gradle features were used in this build, making it incompatible with Gradle 5.0.
> Use '--warning-mode all' to show the individual deprecation warnings.
> See https://docs.gradle.org/4.10.1/userguide/command_line_interface.html#sec:command_line_warnings
> 
> BUILD FAILED in 10s
> 3 actionable tasks: 1 executed, 2 up-to-date

--- a/.gitattributes
+++ b/.gitattributes
@@ -1,0 +1,31 @@
+# Handle line endings automatically for files detected as text
+# and leave all files detected as binary untouched.
+* text=auto
+
+# Never modify line endings of our bash scripts
+*.sh -crlf
+
+#
+# The above will handle all files NOT found below
+#
+# These files are text and should be normalized (Convert crlf => lf)
+*.css           text
+*.gradle        text
+*.html          text
+*.java          text
+*.js            text
+*.json          text
+*.kt            text
+*.md            text
+*.properties    text
+*.txt           text
+*.xml           text
+*.yml           text
+
+# These files are binary and should be left untouched
+# (binary is macro for -text -diff)
+*.class         binary
+*.jar           binary
+*.gif           binary
+*.jpg           binary
+*.png           binary

